src/basic/os-util.c

```diff
@@ -58,7 +58,47 @@ int path_is_extension_tree(const char *path, const char *extension) {
         return 1;
 }
 
-int open_extension_release(const char *root, const char *extension, char **ret_path, int *ret_fd) {
+static int check_extension_release_xattr(int extension_release_fd, char *extension_release_dir_path, char *filename) {
```

Comment:
Could this be renamed so that it's clear what the return value means,
e.g. `extension_release_strict_xattr_value()`? The calling convention for functions that have "check" in the name and return a boolean can be easily misunderstood.